config.js

```diff
@@ -15,3 +15,14 @@ export const defaultSettings = {
 };
 
 // -----------------------------------------------
+
+// API
+export const HOST_API = 'http://localhost:8081';
```

Comment:
You can change this to .env file when you will change localhost.

Reply:
From now will be left for easy access